docs/docs/configuration/ai-tools-config/claude-desktop.md

```diff
@@ -9,3 +9,20 @@ Config**.
 
 Add the `tableau` MCP server to the `mcpServers` object in the config using `config.stdio.json`,
 `config.http.json`, or `config.docker.json` as a template.
+
+## Claude Desktop Extension
+
+Claude Desktop users can also install Tableau MCP as a [Desktop Extension][mcpb]. This is a single
```

Comment:
We should probably use their new terminology. Instead of Desktop Extension it should be MCP Bundle.

Reply:
I believe they are still calling them "Desktop extensions" in a general feature sense, but the file itself is now an MCP Bundle vs Desktop extension. Context: the product UI and web pages like https://www.anthropic.com/engineering/desktop-extensions.